posthog/batch_exports/service.py

```diff
@@ -136,6 +136,7 @@ def backfill_export(batch_export_id: str, start_at: dt.datetime | None = None, e
             BatchExportSchedule's created_at.
     """
     batch_export = BatchExport.objects.get(id=batch_export_id)
+    # TODO data_interval_start and data_interval_end should be optional or handle start_at and end_at being None here
     backfill_run = BatchExportRun.objects.create(
```

Comment:
@hazzadous here too... start_at and end_at are `Optional` but being passed to a Model where they're not. I didn't want to raise a `ValidationError` while having no idea what the code does 🙈

Reply:
Yeah get rid 👍